src/mixs-envo-mapping/mixs-envo-mapping.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""
+Spyder Editor
+
+This is a temporary script file.
+"""
+
+import numpy as np
+import pandas as pds
+import sqlite3
+from datetime import datetime
+
+sql_file = "/home/mam/harmonized_table.db"
```

Comment:
should be a relative path

Reply:
yes, adding `click` functionality today, and will use makefile  (so no more hardcoded paths or settings)